osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -82,7 +81,9 @@ public abstract class DrawableRuleset<TObject> : DrawableRuleset, IProvideCursor
         /// <summary>
         /// The mods which are to be applied.
         /// </summary>
-        private readonly IEnumerable<Mod> mods;
+        [Cached]
+        [Cached(typeof(IBindable<IEnumerable<Mod>>))]
+        private readonly Bindable<IEnumerable<Mod>> mods = new Bindable<IEnumerable<Mod>>();
```

Comment:
What's consuming this bindable here? it seems like in most cases we're not using a bindable any more for mods.

As an aside, I *can* think of scenarios we may want this to be bindable at least here (ie. marathon mode where we may be changing the beatmap / mods but don't want to recreate the ruleset) but this would also mean that a few of these changes away from bindable wouuld likely need changing back in the future (ie. `HUDOverlay`)

Reply:
This case is specifically for the playfield, which now resolves mods.

Reply:
Should it be cached as a non-bindable possibly? What I'm trying to get at is that until now everything has been bindable the whole way, but now these chains are broken in some places and it feels weird caching as bindable here as a result. Like, is there a reason playfield needs it in bindable form (and does it handle `ValueChanged` correctly if so)?

Reply:
Ah I see what you mean. Sure could be non-bindable at this point.